### DISCUSSION TITLE
Alternative to Identity Server is Needed

### DISCUSSION THREAD

**Discussion by 274188A** [abpframework/abp#6309] [General] [answered]
Identity Server will no longer be free / open source after version 4 and the commercial license, at the time of writing this, starts at $1500/year. 

An alternative is needed to Identity Server and Blazor Wasm. 

Maybe the asp.net team is working on this - maybe....

I think we should be considering alternatives soon

**Comment by maliming** [comment, marked as answer]
We are closely following the happenings, no worry :)
